views/post/postView.ejs

```diff
@@ -143,55 +136,123 @@
                         <div class="col my-2">
                             <button onclick='acceptPostScript("delete", postID)' id="deletepost" type="button"
                                 class="btn-universal">Delete</button>
-                            <h2 id="postMessage"></h2>
```

Comment:
Czemu to zostało usunięte? To jest pole na odpowiedź od serwera w razie jakiegoś nieudanego zapostowania.

Reply:
Jest przeniosione niżej do Toast'a